src/components/attendee/RetunableMediaElement/RetunableMediaElement.tsx

```diff
@@ -0,0 +1,90 @@
+import { useCallback, useMemo, useState } from "react";
+
+import { AnyVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+
+import { MediaElement } from "../MediaElement";
+import { useVideoComms } from "../VideoComms/hooks";
+import { VideoSource } from "../VideoComms/types";
+
+import { TuneBanner } from "./components/TuneBanner";
+import { Tuner } from "./components/Tuner";
+import { useRetunableMediaElement } from "./hooks";
+import { RetunableMediaSource } from "./types";
+
+import styles from "./RetunableMediaElement.module.scss";
+
+interface RetunableMediaElementProps {
+  space: WithId<AnyVenue>;
```

Comment:
helps reduce the number of direct imports of `WithId` in a gradual step towards moving it out of `utils/id`

```suggestion
  space: SpaceWithId;
```

Reply:
Resolved in 40c9750b0